sites/fast-tooling-examples/app/style.css

```diff
@@ -1,3 +1,67 @@
+/* This prevents a flash of unstyled components */
+:not(:defined) {
+    visibility: hidden;
+}
+
+:root {
+    --badge-fill-primary: #FB356D;
+    --badge-color-text: white;
+}
+
 body {
     margin: 0;
-}
\ No newline at end of file
+}
+
+.main {
+    display: grid;
+    grid-template-columns: 240px auto;
+    grid-template-rows: 44px auto;
+    min-height: 100vh;
+}
+
+.toolbar {
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 1;
+    grid-row-end: 1;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    border-bottom: 1px solid #e1e1e1;
+}
+
+.toolbar-logo {
+    grid-column-start: 1;
+    grid-column-end: 1;
+}
+
+.toolbar-links {
+    grid-column-start: 2;
+    grid-column-end: 2;
+    text-align: end;
+}
+
+.menu {
+    grid-column-start: 1;
+    grid-column-end: 1;
+    grid-row-start: 2;
+    grid-row-end: 2;
+    border-right: 1px solid #e1e1e1;
```

Comment:
What about `border-inline-end. Seems like we're getting away from the directionSwitch config, and this should work for RTL.